Source/Lib/Common/Codec/EbAvcStyleMcp.h

```diff
@@ -222,72 +222,51 @@ extern "C" {
-                                           uint8_t wpDenominator);
-    void WpAvcStyleLumaInterpolationFilterPosj(EbByte refPic, uint32_t srcStride,
+                                           EbByte tempBuf, uint32_t fracPos);
+    void avc_style_luma_interpolation_filter_posj(EbByte refPic, uint32_t srcStride,
                                            EbByte dst, uint32_t dstStride,
                                            uint32_t puWidth, uint32_t puHeight,
-                                           EbByte tempBuf, uint32_t fracPos,
-                                           int16_t wpWeight, int16_t wpOffset,
-                                           uint8_t wpDenominator);
-    void WpAvcStyleLumaInterpolationFilterPosk(EbByte refPic, uint32_t srcStride,
+                                           EbByte tempBuf, uint32_t fracPos);
+    void avc_style_luma_interpolation_filter_posk(EbByte refPic, uint32_t srcStride,
                                            EbByte dst, uint32_t dstStride,
                                            uint32_t puWidth, uint32_t puHeight,
-                                           EbByte tempBuf, uint32_t fracPos,
-                                           int16_t wpWeight, int16_t wpOffset,
-                                           uint8_t wpDenominator);
-    void WpAvcStyleLumaInterpolationFilterPosp(EbByte refPic, uint32_t srcStride,
+                                           EbByte tempBuf, uint32_t fracPos);
+    void avc_style_luma_interpolation_filter_posp(EbByte refPic, uint32_t srcStride,
                                            EbByte dst, uint32_t dstStride,
                                            uint32_t puWidth, uint32_t puHeight,
-                                           EbByte tempBuf, uint32_t fracPos,
-                                           int16_t wpWeight, int16_t wpOffset,
-                                           uint8_t wpDenominator);
-    void WpAvcStyleLumaInterpolationFilterPosq(EbByte refPic, uint32_t srcStride,
+                                           EbByte tempBuf, uint32_t fracPos);
+    void avc_style_luma_interpolation_filter_posq(EbByte refPic, uint32_t srcStride,
                                            EbByte dst, uint32_t dstStride,
                                            uint32_t puWidth, uint32_t puHeight,
-                                           EbByte tempBuf, uint32_t fracPos,
-                                           int16_t wpWeight, int16_t wpOffset,
-                                           uint8_t wpDenominator);
-    void WpAvcStyleLumaInterpolationFilterPosr(EbByte refPic, uint32_t srcStride,
+                                           EbByte tempBuf, uint32_t fracPos);
+    void avc_style_luma_interpolation_filter_posr(EbByte refPic, uint32_t srcStride,
                                            EbByte dst, uint32_t dstStride,
                                            uint32_t puWidth, uint32_t puHeight,
```

Comment:
Could you also modify the style of the variables, like change dstStride to dst_stride?